packages/cli/src/commands/storybook.js

```diff
@@ -31,9 +31,21 @@ export const builder = (yargs) => {
       type: 'string',
       default: 'public/storybook',
     })
+    .option('manager-cache', {
+      describe:
+        'Cache the manager UI. Disable this when you're making changes to `storybook.manager.js`.',
```

Comment:
Ah this is my bad; I should've suggested double quotes (maybe?)
```suggestion
        "Cache the manager UI. Disable this when you're making changes to `storybook.manager.js`.",
```